requirements.txt

```diff
@@ -1,5 +1,16 @@
 # Core
 pydantic
+pillow
+
+# Linter
+black
+isort
+
+# Testing
+pytest
+
+# Visualization
+matplotlib
```

Comment:
do we wanna make this a part of the core package? normally idgaf but matplotlib is BIG

Reply:
removed